src/Explore.Cli/Program.cs

```diff
@@ -348,10 +366,36 @@ internal static async Task ExportSpaces(string exploreCookie, bool? verboseOutpu
                 ExploreSpaces = spacesToExport
             };
 
+            // set the file name if provided
+            var fileName = string.IsNullOrEmpty(exportFileName) ? "ExploreSpaces.json" : $"{exportFileName}.json";

```

Comment:
We should handle the scenario where folks might append `.json` to the fileName and also error if they append an alternative file extension.